alerts/openshift-container-storage-operator/CephMdsCPUUsageHighNeedsHorizontalScaling.md

```diff
@@ -0,0 +1,51 @@
+# MDSCpuUsageHighNeedsHorizontalScaling
```

Comment:
We should remove the alert 
https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/CephMdsCpuUsageHigh.md,
As we are adding new two alerts

Reply:
@parth-gr ,
I understand your point, but unfortunately all our alerts (in [prometheus-ocs-rules.yaml](https://github.com/red-hat-storage/ocs-operator/blob/main/metrics/deploy/prometheus-ocs-rules.yaml) file) is pointing to this `master` repo files.
So if we remove any existing alerts' runbook entry (from this repo), we won't be able to access it from the `prometheus-ocs-rules.yaml` file (from the other repo) for all the older releases (like 4.17 or 4.16 etc).

That means we can add new alerts, but removal can only be done when the last older release (which points to this runbook) goes out of commision (obselete, no more supported).

Reply:
okay, hope that won't be visible now for 4.18